src/zenml/materializers/base_materializer.py

```diff
@@ -98,6 +98,10 @@ def __new__(
                     associated_type, cls
                 )
 
+            from zenml.utils import notebook_utils
+
+            notebook_utils.save_notebook_cell_id(cls)
```

Comment:
Just in case, maybe `try_save_notebook_cell_id` is a bit more readable?

Reply:
Done